parsec/api/protocol/handshake.py

```diff
@@ -346,7 +353,15 @@ def build_result_req(self, verify_key: Optional[VerifyKey] = None) -> bytes:
             try:
                 answer = self.answer_data["answer"]
                 assert isinstance(answer, bytes)
-                returned_challenge = verify_key.verify(answer)
+
+                # TO-DO remove the else for the next release
```

Comment:
I don't think it's necessary to remove the `else` statement in the next release, since supporting older clients doesn't seem to be a problem in that case (contrary to the newer client/older backend scenario which has a security issue)